### ISSUE TITLE
This package cannot be used in node.js worker thread

### ISSUE THREAD

**Issue by Mykybo** [loretoparisi/fasttext.js#37]
because you use `process.chdir()` it throws error when run inside worker:

`TypeError [ERR_WORKER_UNSUPPORTED_OPERATION]: process.chdir() is not supported in workers`

**Comment by loretoparisi** [comment]
> because you use `process.chdir()` it throws error when run inside worker:
> 
> `TypeError [ERR_WORKER_UNSUPPORTED_OPERATION]: process.chdir() is not supported in workers`

Correct, this can be solved by the way using the WASM version, loaded in a Web Worker. See https://github.com/loretoparisi/fasttext.js/tree/master/wasm
